jsm.go

```diff
@@ -37,6 +37,9 @@ type JetStreamManager interface {
 	// StreamInfo retrieves information from a stream.
 	StreamInfo(stream string, opts ...JSOpt) (*StreamInfo, error)
 
+	// StreamContainedSubjects queries the stream for the subjects it holds with optional filter
+	StreamContainedSubjects(stream string, filter ...string) (map[string]uint64, error)
```

Comment:
Since the code rejects more than 1 filter, not sure why you make it `...string`, and that prevents the use of `opts ...JSOpts` that you should be adding instead. The filter can then technically be made an option.

Reply:
I copied the signature and function name from jsm.go (https://github.com/nats-io/jsm.go/blob/31b0f1e3966a7125454efd34ad99a3267c204e28/manager.go#L564). Not sure why `...string`, just assumed it was done with an eye to allowing more than one filter in the future?

In any case yes it's much better to have the filter in the options